### PR TITLE
Updates to the latest babel configurations (babel-preset-env)

### PR DESCRIPTION
**Pull request by sepandasadi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,5 +80,8 @@
     "include": [
       "src/**/*.@(js|jsx|es6)"
     ]
+  },
+  "dependencies": {
```
Comment (isuttell) [inline package.json]: Is this a dependency or a devDependency?

**Review by isuttell — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sepandasadi) [reply package.json]: Good catch, moved it to devDep

File: .eslintrc [hunk 2]
```diff
@@ -202,7 +202,7 @@
     "react/jsx-max-props-per-line": [1, 1],
     "react/jsx-no-duplicate-props": 1,
     "react/jsx-no-undef": 1,
-    "react/jsx-quotes": 1,
```
Comment (isuttell) [inline .eslintrc:205]: We should switch to the ship components eslint config

**Review by isuttell — APPROVED** [review]

File: .eslintrc (hunk 2 above)
Comment (sepandasadi) [reply .eslintrc:205]: Sure, will do next time I push

**Pull request merged**